addons/folo/common/src/main/java/org/commonjava/indy/folo/conf/FoloConfig.java

```diff
@@ -61,13 +65,44 @@ public Boolean getEnabled()
     {
         return enabled;
     }
-
     @ConfigName( "enabled")
     public void setEnabled( final boolean enabled )
     {
         this.enabled = enabled;
     }
 
+    public boolean isAdminEnabled()
```

Comment:
I think there is no need to add this feature flag for this, that can be done by config in gateway side.

Reply:
So we may need to adjust the endpoints for these query ones to make it different with the monolith one, then we can config the switch on gateway side.

Reply:
Agree. We can control this by gateway. But the recording.enabled is still needed.

Reply:
removed.